client/state/stats/lists/utils.js

```diff
@@ -14,3 +14,52 @@ import toPairs from 'lodash/toPairs';
 export function getSerializedStatsQuery( query = {} ) {
 	return JSON.stringify( sortBy( toPairs( query ), ( pair ) => pair[ 0 ] ) );
 }
+
+/**
+ * Returns a parsed statsPublicize array, ready for use in stats-module
+ *
+ * @param  {Object} data Stats query
+ * @return {Array}       Parsed publicize data array
+ */
+export function statsPublicize( data = {} ) {
+	let response = [];
+	const serviceInfo = {
```

Comment:
I'm sure it's probably not too costly, but instead of creating a new object every time this function is called, maybe move to a top-level constant?